flink-contrib/flink-storm-compatibility/flink-storm-compatibility-core/src/main/java/org/apache/flink/stormcompatibility/wrappers/AbstractStormSpoutWrapper.java

```diff
@@ -97,9 +106,42 @@ public AbstractStormSpoutWrapper(final IRichSpout spout,
 	}
 
 	@Override
+	public void open(Configuration parameters) throws Exception {
+		stormConf = new HashMap();
+
+		/* parameters is task configuration, we can get storm configuration only from job configuration */
+		RuntimeContext ctx = super.getRuntimeContext();
+		if (ctx instanceof StreamingRuntimeContext) {
+			Configuration jobConfiguration = ((StreamingRuntimeContext) ctx).getJobConfiguration();
+
```

Comment:
I see. Would it be better to add Storm config to the task-config to simplify code in `open(...)`  method? For `FlinkTopology` the same config would be added to each Spout/Bolt. At least for embedded mode, using task-config instead of job-config would be more appropriate IMHO.